assignments/array-methods.js

```diff
@@ -55,29 +55,82 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
-console.log(fullName);
+
+//  Study Group Solution
+let fullName = [[runners.first_name] + [runners.last_name]];
+ let result = fullName.map(function(runners) {
+     return [runners.first_name + runners.last_name];
+ })
+ console.log(fullName);
+
+
+
+
+// Q&A Solution
+
+ let fullName=[]
+     fullName = runners.item  {
+     fullName.push('${item.first_name}, $(item.last_name}')
+     return (fullName);
+ }
+
+ console.log(fullName)
```

Comment:
Neither of these are actually correct

all the array methods are doing is replacing the for loop, and giving you the `arrayThisMethodWasCalledOn[i]` that you would normally get from using the for loop. this is what the parameter being handed into your call back function would be. So it would look like this in this case using `forEach` which will execute your call back function on each time it steps to the next index in the array:
```js
// Initialize an empty array that we can push stuff too.
let fullName = []
/* Now we want to populate this array by going through the runners array, and for each 
runner we get as we iterate through we want to run this call back. */
 runners.forEach(runner => { 
    /* Taking the runner that our Array Method passed us here, we are going to target the 
   original array, and push that runners first and last name combined as a string into the 
   next position in the fullName array. */
  fullName.push(runner.first_name + runner.last_name)
} )
```